File: java/src/main/java/net/razorvine/pickle/PickleException.java
package net.razorvine.pickle;

/**
 * Exception thrown when something goes wrong with pickling or unpickling.
 * 
 * @author Irmen de Jong (irmen@razorvine.net)
 */
public class PickleException extends RuntimeException {
	private static final long serialVersionUID = -673995077281654640L;

	public PickleException(String message, Throwable cause) {
		super(message, cause);
	}

	public PickleException(String message) {
		super(message);
	}
}

File: dotnet/Razorvine.Pyrolite/Pyrolite/Pickle/PickleException.cs
/* part of Pyrolite, by Irmen de Jong (irmen@razorvine.net) */

using System;

namespace Razorvine.Pickle
{
	/// <summary>
	/// Exception thrown when something went wrong with pickling or unpickling.
	/// </summary>
	public class PickleException : Exception
	{
		public PickleException(string message) : base(message)
		{
		}

		public PickleException(string message, Exception innerException) : base(message, innerException)
		{
		}
	}
}
